src/test/java/org/octopusden/releng/versions/ComponentVersionFormatMatcherTest.java

```diff
@@ -9,9 +9,9 @@ class ComponentVersionFormatMatcherTest {
 
     private static final ComponentVersionFormatMatcher matcher = new ComponentVersionFormatMatcher(new VersionNames("serviceCBranch", "serviceC", "minorC"));
     private ComponentVersionFormat MODEL_VERSION_FORMAT = ComponentVersionFormat.create("Model.$major.$minor.$service",
-            "Model.$major.$minor.$service.$fix", "Model.$major.$minor.$service.$fix-$build", "Model.$major02.$minor02");
+            "Model.$major.$minor.$service.$fix", "Model.$major.$minor.$service.$fix-$build", "Model.$major02.$minor02", "Model.$major.$minor.$service.$fix");
```

Comment:
hfVersionFormat  value contradicts with following the requirement (from the task):

> Additional validation:
> new validation check is required to ensure hotfixVersionFormat "startsWith" buildVersionFormat (by default buildVersionFormat = releaseVersionFormat)

Reply:
fixed